tests/futhark/test_cca.cpp

```diff
@@ -0,0 +1,53 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2021-2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+// Project include(s).
+#include "traccc/definitions/primitives.hpp"
+#include "traccc/edm/cell.hpp"
+#include "traccc/futhark/cca.hpp"
+
+// Test include(s).
+#include "tests/cca_test.hpp"
+
+// GTest include(s).
+#include <gtest/gtest.h>
+
+// System include(s).
+#include <functional>
+
+namespace {
+traccc::futhark::component_connection cc;
+
+cca_function_t f = [](const traccc::cell_container_types::host &data) {
+    std::map<traccc::geometry_id, vecmem::vector<traccc::measurement>> result;
+
+    traccc::measurement_container_types::host mss = cc(data);
+
+    for (std::size_t i = 0; i < mss.size(); ++i) {
+        vecmem::vector<traccc::measurement> msv;
```

Comment:
You should not rely on the default memory resource with any host container. While (thanks to the polyfill we have in VecMem) this works on all platforms at the moment, we should really use explicit memory resources for all containers.